### ISSUE TITLE
Align Phantom Camera with View

### ISSUE THREAD

**Issue by ramokz** [ramokz/phantom-camera#22]
# Problem 
When instantiating a new PhantomCamera3D, it's set to its default position and rotation. To adjust it, you would either need to set a Follow/Look At property or manually position and rotate it.

The former is not necessarily a desired outcome, and the latter can be a bit tedious to mess around with beyond simple changes.

The proposal here is to set a PhantomCamera3D's position and rotation to the current viewports position and rotation, with the intent to speed up the initial framing process.

# Proposed Solution
Add a button prompt in the property panel for PhantomCamera3D that aligns its positional and rotational value with that of the viewport's.

**Comment by matt-seaton** [comment]
You can align your viewport and press Ctrl+Alt+M or whatever keyboard shortcut you have in your editor settings.

**Comment by ramokz** [comment]
Correct, however, the idea here is to have a slightly more user-friendly approach by having a button in the node's property panel that does just that. Not written above, the follow-up intent of this is to also instantiate the node with the viewport's current transform values, as opposed to at the 0,0 coordinate.

In the grand scheme of things, this is a very minor thing.

**Comment by ramokz** [comment]
Resolved in #600
